_posts/2025-09-30-add-new-table-to-capture-list.adoc

```diff
@@ -0,0 +1,136 @@
+`table.include.list`:: An optional, comma-separated list of regular expressions that match fully-qualified table identifiers for tables whose changes you want to capture. When this property is set, the connector captures changes only from the specified tables. Each identifier is of the form schemaName.tableName. By default, the connector captures changes in every non-system table in each schema whose changes are being captured.
+
+there are also properties to apply opposite logic:
+
+`database.exclude.list`:: An optional, comma-separated list of regular expressions that match the names of databases from which you do not want the connector to capture changes.
+`schema.exclude.list`:: An optional, comma-separated list of regular expressions that match names of schemas for which you do not want to capture changes.
+`table.exclude.list`:: An optional, comma-separated list of regular expressions that match fully-qualified table identifiers for tables whose changes you do not want to capture.
+
+the final configuration of these properties tells Debezium which tables to capture events from.
+
+_How Debezium gets table schema information?_
+
+Debezium, in order to start streaming data changes from a table, needs to know its schema so it can provide information about the table’s structure.
+Normally, Debezium learns about table schemas while streaming by reading DDL statements, parsing them, and updating an in-memory representation of each table’s schema.
+This information is also persisted so that it can survive a connector restart or failure.
+
+For this purpose, there are several properties prefixed with `schema.history.internal.` that manage configuration in this area.
+Of particular interest is the property:
+
+`schema.history.internal.store.only.captured.tables.ddl`:: Specifies whether the connector records schema structures from all tables in a schema or database, or only from tables that are designated for capture.
+
+For now remember it, we will see in detail what role it has later.
+
+Now, let’s take a step back: what happens when a connector is started for the first time?
+
+Debezium requires an initial snapshot for the very same reason.
+Before it can start emitting events for a table, it needs to know the table structure. And even though this information can be obtained from a DDL change while streaming,
+when the connector starts for the very first time it’s quite unlikely that this DDL is still present in the logs.
+This is why Debezium requires the initial snapshot, and you can configure its behavior via the `snapshot.mode` configuration.
+
+As a side note, the initial snapshot is required for this reason, but it can also be useful for snapshotting the current database state — both data and schema — before streaming begins.
```

Comment:
In general the text should be modified a bit. Initial snapshot was introduced to the exisitng data and capturing the schema is the useful by-product.

So capturing the schema is always must, initial snapshot is an extension.